src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -979,11 +1008,174 @@ void testIsPoAttending() {
     @NullAndEmptySource
     void testIsPoAttending(String value) {
         SscsCaseData caseData = SscsCaseData.builder()
-                .dwpIsOfficerAttending(value)
-                .build();
+            .dwpIsOfficerAttending(value)
+            .build();
         boolean result = HearingsDetailsMapping.isPoAttending(caseData);
 
         assertThat(result).isFalse();
     }
 
+    @DisplayName("When DWP is attending then return Face to Face as preferred hearing type")
+    @Test
+    void whenDwpIsAttending_thenReturnFaceToFace_asPreferredHearingType() {
+        SscsCaseData caseData = SscsCaseData.builder()
+            .dwpIsOfficerAttending(YesNo.YES.getValue())
+            .appeal(Appeal.builder()
+                        .hearingType(HearingChannel.FACE_TO_FACE.getHmcReference())
+                        .build())
+            .build();
+        List<HearingChannel> result = getHearingChannel(caseData);
+        assertEquals(HearingChannel.FACE_TO_FACE, result.get(0));
```

Comment:
(Minor0 try to prefer assertJ or JUnit5 assertions, they are much better. You should also assert that a list has an expected size before moving on to assert the values

Reply:
Added